.ci.yaml

```diff
@@ -320,6 +320,10 @@ targets:
     properties:
       config_name: mac_ios_engine
       environment: Staging
+      dependencies: >-
```

Comment:
@CaseyHillers does this overrides the property or expands it?

Reply:
This extends it. If the platform defines jazzy with a version, this will do nothing.

So if you want Mac dependencies + jazzy, this is the correct change.

Reply:
Yes, that's what was looking for. Thanks!